Specs/DateTime/English/DateTimeModel.json

```diff
@@ -10592,5 +10592,125 @@
         }
       }
     ]
+  },
+  {
+    "Input": "Please can you arrange a Microsoft Teams Meeting starting on January 7th to discuss ARM Templates?",
```

Comment:
Such cases already worked in .Net with just the regex change? Why do the other plats need the if logic change? To be in sync with .Net or are the implementations diverging?

Reply:
Yes, the logic in .NET and other platforms are different in the previous version.